lib/ace/mode/latex_parser.js

```diff
@@ -1,30 +1,59 @@
 define(function(require, exports, module) {
 "use strict";
 
-var WORD_PATTERN = "[a-zA-Zа-яА-ЯёЁ]+";
-var WORD_REGEX = new RegExp("^" + WORD_PATTERN);
+var WORD_REGEX = /^[a-zа-яё]+/i,
+    // % * <foo@bar.com> 2015-01-12T00:17:27.829Z:
+    DISCUSSION_COMMENT_REGEXP = /^\s*%\s([\*\^])\s<(.+)>(\s(.+))?:$/,
+    DISCUSSION_TEXT_ROW_REGEXP = /^\s*%\s*(.*)/;
```

Comment:
I match the whole line of text with this regex, so it is regular tex comment without other tokens before.